client/my-sites/checkout/composite-checkout/components/wp-order-review-line-items.tsx

```diff
@@ -646,6 +646,115 @@ function AnnualDiscountCallout( {
 	return null;
 }
 
+function IntroductoryOfferCallout( {
+	product,
+}: {
+	product: ResponseCartProduct;
+} ): JSX.Element | null {
+	const translate = useTranslate();
+	let text: TranslateResult = translate( 'Discount for first period' );
+	if ( ! product.introductory_offer_terms?.enabled ) {
+		return null;
+	}
+
+	const intervalUnit = product.introductory_offer_terms.interval_unit;
+	const intervalCount = product.introductory_offer_terms.interval_count;
+
+	if ( product.cost === 0 ) {
+		if ( intervalUnit === 'day' ) {
+			if ( intervalCount === 1 ) {
+				text = translate( 'Free for first day' );
+			} else {
+				text = translate( 'Free for first %(numberOfDays)d days', {
+					args: {
+						numberOfDays: intervalCount,
+					},
+				} );
+			}
+		}
+		if ( intervalUnit === 'week' ) {
+			if ( intervalCount === 1 ) {
+				text = translate( 'Free for first week' );
+			} else {
+				text = translate( 'Free for first %(numberOfWeeks)d weeks', {
+					args: {
+						numberOfWeeks: intervalCount,
+					},
+				} );
+			}
+		}
+		if ( intervalUnit === 'month' ) {
+			if ( intervalCount === 1 ) {
+				text = translate( 'Free for first month' );
+			} else {
+				text = translate( 'Free for first %(numberOfMonths)d months', {
+					args: {
+						numberOfMonths: intervalCount,
+					},
+				} );
+			}
+		}
+		if ( intervalUnit === 'year' ) {
+			if ( intervalCount === 1 ) {
+				text = translate( 'Free for first year' );
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **23** times:
`translate( 'First year free' )` *ES Score: 6*